frontend/src/components/productGrid/CategoryIndex.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './ProductGridIndex.css';
+import GenerationImage from 'assets/dummyImages/powerGeneration.jpg';
+import TransmissionImage from 'assets/dummyImages/powerTransmission.jpg';
+import IndustrialImage from 'assets/dummyImages/Industrialapplications.jpg';
+
+function CategoryIndex() {
+    return (
+        <div className='w3-row w3-padding '>
+            <div className='w3-col w3-padding-small w3-padding-top'>
+                <strong className='SubTitle'>Our Offerings</strong>
+                <p className='TextContent'>
+                    Meeting the growing global energy demand while transitioning to a more
+                    sustainable world requires a complex journey. But you're not without a partner.
+                    At Siemen's Carbon Visualization, we work closely with our customers to drive
+                    the energy transition, step by step, with our rich history of innovative
+                    technology,extensive energy experience and ambitious strategy to drive the
+                    decarbonization of global energy systems.
+                </p>
+            </div>
+            <div class='w3-row m6 32 w3-padding-small'>
+                <div class='w3-third imageTitle w3-animate-bottom'>
```

Comment:
all these `class` should be `className` 🙂  @manifau 

have a look at the console (F12) There it says

![image](https://user-images.githubusercontent.com/22003632/122614454-f6d0b480-d086-11eb-967a-5f2976aaeeeb.png)

🙂

Reply:
Thanks for pointing it out. I will change them :)

Reply:
thank you. you are welcome :)